src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,19 @@
+import { jwtService } from '../services/jwt.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    return res.sendStatus(401);
+  }
+
+  const userData = jwtService.verifyToken(token);
```

Comment:
Consider adding error handling for the `verifyToken` method. If the token is invalid or expired, `verifyToken` might throw an error, which should be caught to prevent the application from crashing.